layers/+lang/java/README.org

```diff
@@ -117,9 +117,12 @@ at the root of your project first.
 | ~SPC m g d~   | Goto type definition                     |
 | ~SPC m g a~   | Search type in project                   |
 | ~SPC m g A~   | Search type in project using ~lsp-ui~    |
+| ~SPC m g k k~ | Show type hierarchy                      |
```

Comment:
tb is missing

Reply:
Also IMHO, type hierarchy should be under `,gh`

Reply:
will add `t b`, `,gh` is used by call hierarchy

Reply:
then `,gH`?

Reply:
I'd prefer lower case letters. imo bindings having upper case letter is a pain to invoke.